example/dj/apps/issue_tracker/cores/__init__.py

```diff
@@ -75,6 +80,7 @@ def get_rest_patterns(self):
         )
         return rest_patterns
 
+    @short_description('count created issues')
```

Comment:
```suggestion
    @short_description('created issues count')
```
Is better from language point of view.